llvm/lib/SYCLLowerIR/SYCLPropagateAspectsUsage.cpp

```diff
@@ -504,12 +508,17 @@ buildFunctionsToAspectsMap(Module &M, TypeToAspectsMapTy &TypesWithAspects,
                            const AspectValueToNameMapTy &AspectValues,
                            const std::vector<Function *> &EntryPoints) {
   FunctionToAspectsMapTy FunctionToAspects;
+  // The set of spects from FunctionsWithDeclaredAspects should be merged to the set
```

Comment:
I suggest that we move this comment down below, right before we do the actual merging

Reply:
Sure [ae09ca5](https://github.com/intel/llvm/pull/7419/commits/ae09ca51ce4ecb0249d0156c26b7c766cf03df27)